locale/dbr/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,11250 @@
+msgstr "‮⊥ɥǝ ɯısɔǝʅʅɐuǝons ɔɐʇǝƃoɹʎ ɔɐuuoʇ qǝ ɔoɯqıuǝp ʍıʇɥ ɐppıʇıouɐʅ ɔɐʇǝƃoɹıǝs˙"
+
+#: src/olympia/addons/forms.py
+#, python-format
+msgid "Before changing your default locale you must have a name, summary, and description in that locale. You are missing %s."
+msgstr "‮Ԑǝɟoɹǝ ɔɥɐuƃıuƃ ʎonɹ pǝɟɐnʅʇ ʅoɔɐʅǝ ʎon ɯnsʇ ɥɐʌǝ ɐ uɐɯǝ´ snɯɯɐɹʎ´ ɐup pǝsɔɹıdʇıou ıu ʇɥɐʇ ʅoɔɐʅǝ˙ ⅄on ɐɹǝ ɯıssıuƃ %s‮˙"
+
+#: src/olympia/addons/forms.py
+msgid "A license must be selected."
+msgstr "‮∀ ʅıɔǝusǝ ɯnsʇ qǝ sǝʅǝɔʇǝp˙"
+
+#: src/olympia/addons/forms.py
+msgid "Give Your Theme a Name."
+msgstr "‮⅁ıʌǝ ⅄onɹ ⊥ɥǝɯǝ ɐ Nɐɯǝ˙"
+
+#: src/olympia/addons/forms.py src/olympia/devhub/templates/devhub/personas/submit.html
+msgid "Describe your Theme."
+msgstr "‮ᗡǝsɔɹıqǝ ʎonɹ ⊥ɥǝɯǝ˙"
+
+#: src/olympia/addons/forms.py
+msgid "Enter a new author's email address"
+msgstr "‮Ǝuʇǝɹ ɐ uǝʍ ɐnʇɥoɹ,s ǝɯɐıʅ ɐppɹǝss"
+
+#: src/olympia/addons/helpers.py
+msgid "Not Reviewed"
+msgstr "‮Noʇ ᴚǝʌıǝʍǝp"
+
+#: src/olympia/addons/models.py
+msgid "Users have the option of contributing more or less than this amount."
+msgstr "‮∩sǝɹs ɥɐʌǝ ʇɥǝ odʇıou oɟ ɔouʇɹıqnʇıuƃ ɯoɹǝ oɹ ʅǝss ʇɥɐu ʇɥıs ɐɯonuʇ˙"
+
+#: src/olympia/addons/models.py
+msgid "Users will always be asked in the Add-ons Manager (Firefox 4 and above). Only applies to desktop."
+msgstr "‮∩sǝɹs ʍıʅʅ ɐʅʍɐʎs qǝ ɐsʞǝp ıu ʇɥǝ ∀pp-ous Wɐuɐƃǝɹ (Ⅎıɹǝɟox 4‮ ɐup ɐqoʌǝ)˙ Ouʅʎ ɐddʅıǝs ʇo pǝsʞʇod˙"
+
+#: src/olympia/addons/models.py src/olympia/constants/base.py
+msgid "Listed"
+msgstr "‮⅂ısʇǝp"
+
+#. l10n: {0} is the addon name
+#: src/olympia/addons/views.py
+msgid "Contribution for {0}"
+msgstr "‮Ↄouʇɹıqnʇıou ɟoɹ {0}"
+
+#: src/olympia/addons/views.py
+msgid "Abuse reported."
+msgstr "‮∀qnsǝ ɹǝdoɹʇǝp˙"
+
+#: src/olympia/addons/widgets.py
+msgid "My add-on doesn't fit into any of the categories"
+msgstr "‮Wʎ ɐpp-ou poǝsu,ʇ ɟıʇ ıuʇo ɐuʎ oɟ ʇɥǝ ɔɐʇǝƃoɹıǝs"
+
+#: src/olympia/addons/templates/addons/button.html src/olympia/addons/templates/addons/impala/button.html
+msgid "No compatible versions"
+msgstr "‮No ɔoɯdɐʇıqʅǝ ʌǝɹsıous"
+
+#. {0} is a platform name like Windows or Mac OS X.
+#: src/olympia/addons/templates/addons/button.html src/olympia/addons/templates/addons/includes/install_button.html
+msgid "for {0}"
+msgstr "‮ɟoɹ {0}"
+
+#: src/olympia/addons/templates/addons/button.html
+msgid "View privacy policy"
+msgstr "‮Ʌıǝʍ dɹıʌɐɔʎ doʅıɔʎ"
+
+#: src/olympia/addons/templates/addons/button.html src/olympia/editors/templates/editors/addon_details_box.html src/olympia/legacy_discovery/templates/legacy_discovery/addons/detail.html
+msgid "View End-User License Agreement"
+msgstr "‮Ʌıǝʍ Ǝup-∩sǝɹ ⅂ıɔǝusǝ ∀ƃɹǝǝɯǝuʇ"
+
+#: src/olympia/addons/templates/addons/button.html src/olympia/addons/templates/addons/impala/button.html
+#, python-format
+msgid "This add-on has not been reviewed by Mozilla. <a href=\"%(url)s\">Learn more</a>"
+msgstr "‮⊥ɥıs ɐpp-ou ɥɐs uoʇ qǝǝu ɹǝʌıǝʍǝp qʎ Wozıʅʅɐ˙ <a href=\"%(url)s\">‮⅂ǝɐɹu ɯoɹǝ</a>"
```

Comment:
I had started writing a lib to fix this precise problem but hadn't gotten around to completing it. Until we can reliably process html as RTL I think we should leave `dbr` out completely.